src/index.html

```diff
@@ -18,7 +18,7 @@
         </div>
         <ul class="nav-menu d-flex">
           <li class="nav-item"><a href="#categories">14 Categories</a></li>
-          <li class="nav-item"><a href="#meals"><span class="item-count">16</span> Meal(s) found!</a></li>
+          <li class="nav-item"><a href="#meals"><span class="item-count"><!--Dyanamic counter--></span> Meal(s) found!</a></li>
```

Comment:
could we remove `<!--Dyanamic counter-->`? for cleaner code

Reply:
Alright Gotcha 💯